tests/dummy/app/templates/application.hbs

```diff
@@ -133,11 +133,32 @@
     {{github-issue title="Focus Ordering"}}
 
     <h3>Query Params</h3>
-    <p>Scenarios to write:</p>
-    <ul>
-      <li>Refreshing model</li>
-      <li>Non-refreshing model</li>
-    </ul>
+    <p>These tests are for focusing behavior. You may use your mouse to click through the links in this test.</p>
+    <ol>
+      <li><button {{action "resetDemo"}}>Reset Demo</button></li>
+      <li>{{#link-to "feed"}}Load the Feed route.{{/link-to}}</li>
+      <li>Focus should be set to a block which begins with the "Feed" header.</li>
+
+      <li>{{#link-to "feed" (query-params sort="title")}}Update query params to sort by title.{{/link-to}}</li>
+      <li>Model was <em>not</em> refreshed; full transition <em>not</em> performed.</li>
+      <li>Items should reorder into alphabetical order.</li>
+      <li>No focus should be set.</li>
```

Comment:
I'm not 100% sure if this is the behavior we want. @ember-a11y/core Do y'all have opinions?